docs/Migrating-v3.md

```diff
@@ -116,6 +116,20 @@ Example: `@ecl-twig/ec-component-accordion/ecl-accordion.html.twig` => `@ecl/acc
 
 [ECL Compliance component](https://github.com/ec-europa/ecl-twig/tree/master/src/ec/packages/ec-component-ecl-compliance) has been removed.
 
+## Presets / themes
+
+A new concept of "theme" has been introduced, which complete the presets used in ECL 2. Presets are still used under the hook (one for each theme), but what is exposed to end users is now the following:
```

Comment:
```suggestion
A theme layer has been introduced in ECL v3. It sits on top of the components layer and transforms variables into values targeting specific use cases called presets. Presets are ready-made ECL releases serving for different purposes and target audiences. The following presets are available:
```

Reply:
Reading the overall text makes me thing whether it's worth putting information regarding themes here. There's nothing essential and the focus is vice versa: it's on the presets and the themes is under the hood.